src/app/loadout-builder/generated-sets/GeneratedSetItem.tsx

```diff
@@ -120,10 +136,26 @@ export default function GeneratedSetItem({
         />
       )}
       {$featureFlags.armor2ModPicker && (
-        <div className={'lockedMods'}>
-          {lockedMods?.map((mod) => (
-            <SocketDetailsMod key={mod.mod.hash} itemDef={mod.mod} defs={defs} />
-          ))}
+        <div className={styles.lockedSockets}>
+          {Boolean(lockedMods?.length) && (
+            <div className={`lockedItems ${styles.lockedSocketsRow}`}>
+              {lockedMods?.map((mod) => (
+                <SocketDetailsMod key={mod.mod.hash} itemDef={mod.mod} defs={defs} />
+              ))}
+            </div>
+          )}
+          {Boolean(lockedOldMods.length) && (
+            <div className={`lockedItems ${styles.lockedSocketsRow}`}>
```

Comment:
We've been using `clsx` for this sort of thing.

Reply:
Cool, I will clean it up in the next one